config.py.in

```diff
@@ -145,6 +149,10 @@ class KimchiConfig(PluginConfig):
                                  'path': kimchiPaths.spice_file},
             '/spice-html5/spice.css': {'type': 'file',
                                        'path': kimchiPaths.spice_css_file},
+            '/spice-web-client': {'type': 'dir',
+                                  'path': kimchiPaths.spice_dir},
+            '/index.html': {'type': 'file',
```

Comment:
I don't think we need to expose `index.html`. It will be inside the dir `/spice-web-client` and will be accessible by ``/spice-web-client/index.html`.

Reply:
That's correct, fixed!